modules/odc-adding-services-to-application.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc
+
+[id="odc-adding-services-to-your-application_{context}"]
+= Adding services to your application
+
+In the *Topology view*, access the *+Add* actions using the context menu, sidebar, or by hovering and dragging the dangling arrow on to the *Topology view* to add a service to your application.
+
+.Procedure
+
+To add a service using the context menu:
```

Comment:
```suggestion
```

Reply:
The IBM Style Guide or Red Hat SSG recommends avoiding this type of sentence. The information it contains is redundant with the title and the contents of the procedure.